### PR TITLE
[GH-#164] Sync transaction pool

### LINKED ISSUE #164: Sync transaction pool

### PR DESCRIPTION
**Pull request by d-velev**
closes #164

### CODE REVIEW

File: apps/aecore/lib/aecore/peers/sync.ex [hunk 1]
```diff
@@ -39,6 +40,17 @@ defmodule Aecore.Peers.Sync do
     GenServer.call(__MODULE__, :add_valid_peer_blocks_to_chain)
   end
 
+  def add_unknown_peer_pool_txs(peers) do
+    Enum.each(peers, fn(peer) ->
+        deserialized_pool_txs =
+          peer
+          |> HttpClient.get_pool_txs()
+          |> elem(1)
+          |> Enum.map(fn(tx) -> Serialization.tx(tx, :deserialize) end)
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/peers/sync.ex]: Deserialization should be handled by `HttpClient`.

File: apps/aecore/lib/aecore/peers/worker.ex [hunk 2]
```diff
@@ -224,6 +224,8 @@ defmodule Aecore.Peers.Worker do
               Logger.info(fn -> "Added #{uri} to the peer list" end)
               Sync.ask_peers_for_unknown_blocks(updated_peers)
               Sync.add_valid_peer_blocks_to_chain()
+              Sync.add_unknown_peer_pool_txs(updated_peers |> Map.values()
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/peers/worker.ex]: Please move extraction of uri-s to `Sync.add_unknown_peer_pool_txs` as in other functions.

File: apps/aecore/lib/aecore/peers/sync.ex (hunk 1 above)
Comment (thepiwo) [reply apps/aecore/lib/aecore/peers/sync.ex]: correct, thanks!

**Comment by thepiwo** [comment]
@d-velev can you merge with master?

**Pull request merged**